src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -38,7 +38,43 @@
     // FIXME: Copy this value to the new key
     // The original key for this value was "Shoot From Wall Hood Setpoint"
     // The new key is "SHOOT_FROM_WALL_HOOD_SETPOINT"
-    SHOOT_FROM_WALL_HOOD_SETPOINT(60);
+    SHOOT_FROM_WALL_HOOD_SETPOINT(60),
+
```

Comment:
Why all the extra blank lines?

Reply:
The automatic formatter tries to fit the declarations onto as few lines as possible. Adding the blank lines keeps each declaration on its own line. IMO, this is the more readable of the two options.